src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -777,9 +907,9 @@ public void tick() {
             if (progress < 0) return;
             if (progress == 0 || --progress == 0) {
                 // id==0 will be end of chain if 1 input, so we need a +1 here
-                if (id + 1 >= currentRecipe.mInputs.length) {
-                    addOutput(currentRecipe.mOutput);
-                    reset();
+                if (id + 1 >= currentInputLength) {
+                    if (addOutput(currentRecipe.mOutput) || !voidingMode.protectItem) reset();
```

Comment:
This is not enough for void protection, the machine should not start if it can't output. There's helper method `canOutputAll`, but I'm not sure if this multi can adjust void protection well

Reply:
this machine is unique in it that it has a STUCKED state, so it can just hang there (without draining power) while waiting for the user/automation to fix the output place. While this might break the consistency across other multis in how void protection works, I feels this is the bette way to handle this mechanism here.

Reply:
I see